src/components/HeaderBar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import '../assets/styles/headerBar.scss'
+import { Route } from 'react-router-dom'
+
+const HeaderBar = () => {
+    return (
+        <>
```

Comment:
Do we need that fragment here? Or we can just use the inner div with className="header" as a root element?